static-site/src/components/ListResources/Showcase.jsx

```diff
@@ -10,18 +12,35 @@ const cardWidthHeight = 160; // pixels
 
 export const Showcase = ({cards, setSelectedFilterOptions}) => {
   if (!cards.length) return null;
+
+  // Generate options for carousel
+  let responsiveOptions = {};
+
+  for (let i = 0; i < cards.length; i++) {
+    const breakpointMin = i * cardWidthHeight;
+    let breakpointMax = (i + 1) * cardWidthHeight;
+
+    // Don't limit max otherwise carousel won't render at all on larger screens
+    if (i == cards.length - 1) {
+      breakpointMax = 999999
+    }
+
+    responsiveOptions[i] = {
+      breakpoint: { max: breakpointMax, min: breakpointMin },
+      items: i
+    };
+  }
```

Comment:
@tsibley re: https://github.com/nextstrain/nextstrain.org/pull/873#issuecomment-2123601423

> There's some non-ideal behaviour going on with the responsive widths of the tiles at some viewport widths

Unfortunately react-multi-carousel takes a fixed number of items opposed to a fixed width per item, which I would prefer avoids the behavior you'r seeing. With a fixed number of items, there's bound to be some stretching and contracting. I did a quick and dirty calculation here – will tweak it to contract a bit less.

Reply:
Can you use a flexbox to evenly space them? That seems better than stretching / contracting cards

Reply:
I presume the bigger issue behind this is that we're not observing / responding to viewport changes (this is from memory... we do observe this elsewhere in the resource listing UI).

Reply:
Scratch that, the viewport observation is flowing into the carousel. But if we've scrolled (?) to the end of the carousel and then expand the page it doesn't respond appropriately. I'd consider this minor / don't need to fix:

<img width="1174" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/2548ca0c-1889-4e93-9370-c098fea69619">


A separate issue is the number of cards displayed for a given width, which is causing them to become rectangles:

<img width="758" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/457a7ae5-1aa9-4da0-ad85-b8103a1dfd97">